### ISSUE TITLE
Extract validator to CLI

### ISSUE THREAD

**Issue by OliverJAsh** [getsentry/sourcemaps.io#23]
It would be fantastic if you could extract the validator part of this service out into a CLI, so it can be used without needing files to be internet accessible

There is this project https://github.com/ben-ng/sourcemap-validator, however I get vastly different results when using this.

**Comment by danschultzer** [comment]
I agree. I would love to be able to hook it up in CI to ensure no source map issues.

**Comment by danschultzer** [comment]
Just saw that `sentry-cli` does it, though it's only during upload: https://github.com/getsentry/sentry-cli/blob/f1e2073fc9ba048f845cfda6161edc558d23a4e4/docs/releases.rst#managing-release-artifacts

**Comment by benvinegar** [comment]
Hey, totally agree – and somehow I missed this issue months ago 😅

**Comment by krutoo** [comment]
Hi, is it possible to validate source map by CLI without upload?
